content/sse/index.textile

```diff
@@ -42,6 +42,64 @@ If using basic auth, you can use a querystring parameter of @key@ or an @Authori
 
 Connection state is only retained for two minutes. See "Connection state explained":/realtime/connection#connection-state-explained for full documentation.
 
+The SSE protocol and the EventSource API are designed so that a dropped connection is resumed transparently; the client implementation will reconnect and supply a @lastEventId@ param that ensures that the resuming connection delivers any events that have arisen since the connection was dropped. Ably uses this mechanism to reattach all channels in a new connection to the exact point that had been reached in the prior connection.
+
+When a token expires the connection will end. However, the default EventSource behaviour of automated reconnection will not work, because the (expired) credentials are part of the connection URL. What is needed is for a new connection to be established, with an updated @accessToken@. The question then arises as to how to do that with continuity - that is, how to establish a new connection but supply the correct @lastEventId@ so that the new connection resumes from the point that the prior connection became disconnected.
+
+
+h3(#message-continuity-token-auth). Implementing message continuity with Token Auth
+
+Implementing transparent connection resumes when tokens need to be renewed requires a few additional steps - detecting token expiry and resuming the connection from the point of the last delivered message using the @lastEventId@ attribute:
+
+h4(#detecting-token-expiry). Detecting token expiry
+
+When a connection is closed as a result of any error (that is, it's not just a dropped connection), then the @onerror@ event will occur on the EventSource instance, and the data attribute of the event will contain an Ably error body with the information about the nature of error. In the case of a token error - that is an error arising from a problem with the auth token - the code in the error body will indicate that. Token errors have a code in the range @40140 <= code < 40150@. In such cases, the authentication can be retried with a new @accessToken@.
```

Comment:
> `the @onerror@ event`

the event name is `error`; `onerror` is just a way of adding a listener that fires for an `error` event

Reply:
You are right. Fixed, thanks